docs.md

```diff
@@ -0,0 +1,1366 @@
+
+This documentation covers the technical part of writing only, i.e. how to use our custom DART templates and macros, and how to make sure a module you're writing will pass our Quality Assurance process. 
+
+For more general guidance about how to contribute to the DART project, including how to propose a new module, see our [contributing guidelines](https://github.com/arcus/education_modules/blob/templates_update/CONTRIBUTING.md). 
+Also, before you begin writing a new module for the first time, be sure to look at some of our [existing modules](https://arcus.github.io/education_modules/list_of_modules) for examples of formatting, style, and tone.
+
+If you're new to markdown, check out this general [guide to markdown syntax](https://daringfireball.net/projects/markdown/syntax). 
+
+Most things that work in regular markdown will also work in LiaScript, but there are a few exceptions. 
+To see how markdown works in LiaScript, look over the [LiaScript documentation on markdown syntax](https://liascript.github.io/course/?https://raw.githubusercontent.com/liaScript/docs/master/README.md#markdown-syntax).
+
+<div class = "behind-the-scenes">
+<b style="color: rgb(var(--color-highlight));">Behind the scenes</b><br>
+
+This document is itself a LiaScript markdown file! 
+To see what the text looks like, open the [raw version of this file](https://raw.githubusercontent.com/arcus/education_modules/roseh_docs/docs.md).
+
+</div>
+
+## Module review process
+
+When you write a new DART module, it will go through a formal review process before being published, called Quality Assurance (QA). 
+The instructions in this documentation will help you make sure your module will pass QA with as few revisions as possible.
+
+<div class = "behind-the-scenes">
+<b style="color: rgb(var(--color-highlight));">Behind the scenes</b><br>
+
+Our QA process is very transparent. 
+You can view the [QA checklist](https://github.com/arcus/education_modules/blob/templates_update/.github/ISSUE_TEMPLATE/qa-for-standard-modules.md) that will be used to check your module at any time. 
+You can also see revisions and comments on [pull requests for other modules that have already been QAed](https://github.com/arcus/education_modules/pulls?q=is%3Apr+is%3Aclosed+label%3A%22Quality+Assurance%22) to see what kinds of adjustments were made. 
+
+</div>
+
+## Style and formatting requirements
+
+- The title should be the only level-1 header in the module document.
+- LiaScript will create a new page at each level 1, 2, or 3 header, so to avoid a page with only a header and no content, include text after each header before the next.
+- We have a standardized naming convention and directory structure:
+
+  * Folder and file names use lowercase and underscores (no dashes)
+  * Main module directory folder name is identical to the name of the module content markdown file.
+  * Images, videos, and other audio-visual assets are saved within a `media` folder within the module directory
```

Comment:
Some modules also have a `data` folder for jupyter notebooks that is separate from the `media` folder, see pandas_transform and elements_of_maps. It may be that these modules need to be updated to have the correct file structure, but it may also be useful to have both a `media` and a `data` file like elements_of_maps has.

Reply:
Oh, yeah, let's add a folder for that! Is it always code/notebooks? Because maybe `data` isn't the right name, then (maybe `src`?) This would be a sensible moment to change that convention, if we want.

Reply:
`src` seems like a great option for that, encompassing all sorts of files from raw data to jupyter notebooks etc.